src/lib/libc_preload/syscall_wrappers.c

```diff
@@ -240,6 +235,9 @@ INTERPOSE(ftruncate);
 #ifdef SYS_futex // kernel entry: num=202 func=sys_futex
 INTERPOSE(futex);
 #endif
+#ifdef SYS_futex_waitv // kernel entry: num=449 func=sys_futex_waitv
+INTERPOSE(futex_waitv);
+#endif
```

Comment:
I don't see a `futex_waitv` libc function, so maybe this shouldn't be included here. (And the same for `set_mempolicy_home_node` below).